src/edu/wright/cs/raiderplanner/model/Extension.java

```diff
@@ -45,39 +45,71 @@ protected void replace(VersionControlEntity receivedVce) {
 		super.replace(receivedVce);
 	}
 
+	/**
+	 * Enumerated values for approval statuses.
+	 * @author unknown
```

Comment:
The Git history will allow you to see who authored this code. Please have a look and update this comment.

Reply:
How can I find it in the git history? I've undone the accidental changes to the metadata, just need to do this now.

Reply:
In the CLI, you can do `git blame path/to/file` and then look for the specific lines. In Eclipse, right click on the line number and select `Show Revision Info`. Then you hover over the line number to see who introduced the line and when.